marklogic-data-hub-central/ui-custom/src/components/confirmation-modal/confirmation-modal.scss

```diff
@@ -0,0 +1,18 @@
+button.submit {
```

Comment:
Can we wrap all these styles in a .confirmation-modal wrapper to avoid pollution in other widgets? Does that make sense? (See other comment below.)